marathon-template

```diff
@@ -20,14 +21,27 @@ def http(url):
 def fetch_tasks(marathon_uri, app_id):
   app = http(marathon_uri+"/v2/apps/"+app_id)
   app_tasks = app["app"]["tasks"]
-  def task_info(host, ports):
+  app_labels = app["app"]["labels"]
+  def task_info(host, ports, labels):
     try:
-      return MarathonTaskInfo(host, ports)
+      return MarathonTaskInfo(host, ports, labels)
     except:
       return None
-  tasks = filter(lambda x: x is not None, map(lambda task: task_info(task["host"], task["ports"]), app_tasks))
+
+  tasks = filter(lambda x: x is not None, map(lambda task: task_info(task["host"], task["ports"], app_labels), app_tasks))
   return sorted(tasks, key=lambda task: task.name())
 
+def apps(marathon_uri):
+  apps = http(marathon_uri+"/v2/apps/")
```

Comment:
I see we're pushing 2 changes as part of the same PR. @lakshmirajagopalan It would be great if we can push just a single change as part of the PR.